examples/s2i-nginx/assemble

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+#
+# S2I assemble script for the 'nginx-centos7' image.
+# The 'assemble' script builds your application source ready to run.
+#
+# For more information refer to the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+
+if [ "$1" = "-h" ]; then
```

Comment:
i understand, and i will change them. however i kept the templates that are created by `s2i create`. so those need to be changed. i will probably submit a pull request with your suggestion a bit later on :)